admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -158,54 +149,15 @@ export const ApplicationPage: React.FunctionComponent = () => {
-                                                    id='name'
-                                                    name="name"
-                                                /></FormGroup>
-                                            <FormGroup label='Display name' fieldId='display-name' isRequired
-                                                helperText='This is the name you want to display on the UI'>
-                                                <TextInput
-                                                    type='text'
-                                                    value={ eventType.displayName }
-                                                    onChange={ handleChange }
-                                                    id='display-name'
-                                                    name="displayName"
-                                                /></FormGroup>
-                                            <FormGroup label='Description' fieldId='description'
-                                                helperText='Optional short description that appears in the UI
-                                                to help admin descide how to notify users.'>
-                                                <TextArea
-                                                    type='text'
-                                                    value={ eventType.description }
-                                                    onChange={ handleChange }
-                                                    id='description'
-                                                    name="description"
-                                                /></FormGroup>
-                                            <ActionGroup>
-                                                <Button variant='primary' type='submit'
-                                                    { ...(newEvent.loading || newEvent.payload?.status !== 200) ?
-                                                        <Spinner /> : eventTypesQuery.payload.value }
-                                                    onClick={ handleSubmit }>{isEdit ? 'Update' : 'Submit' }</Button>
-                                                <Button variant='link' type='reset'
-                                                    onClick={ () => setShowModal(false) }>Cancel</Button>
-                                            </ActionGroup>
-                                        </Form>
-                                    </Modal>
+                                    <CreateEditModal
+                                        isEdit={ isEdit }
+                                        showModal={ showModal }
+                                        applicationName={ application?.displayName }
+                                        eventType={ eventType }
+                                        onClose={ onClose }
```

Comment:
This `onClose` function sets the `showDeleteModal` to false, not the `showModal`.
I suggest to create one `onCloseDeleteModal` and one `onCloseCreateEditModal` or something similar.